src/tests/class_tests/openms/source/AverageLinkage_test.cpp

```diff
@@ -157,6 +157,7 @@ START_SECTION((static ClusterFunctor* create()))
 {
 	ClusterFunctor* cf = AverageLinkage::create();
   TEST_NOT_EQUAL( dynamic_cast<AverageLinkage*>(cf) , nullPointer)
+  delete cf;
```

Comment:
RAII with unique_ptr?

Reply:
I dont think it is worth for the tests where you delete 2 lines after. Feel free to do it if you think it is important.

Reply:
my argument would be: someone forgot it here :P